src/node_errors.cc

```diff
@@ -195,53 +196,85 @@ void PrintStackTrace(Isolate* isolate, Local<StackTrace> stack) {
 
     if (stack_frame->IsEval()) {
       if (stack_frame->GetScriptId() == Message::kNoScriptIdInfo) {
-        FPrintF(stderr, "    at [eval]:%i:%i\n", line_number, column);
+        char buf[64];
+        snprintf(
+            buf, sizeof(buf), "    at [eval]:%i:%i\n", line_number, column);
+        result += std::string(buf);
```

Comment:
You could probably make all of these

```suggestion
        result += SPrintF("    at [eval]:%i:%i\n", line_number, column);
```

etc. if you want to